.github/workflows/screenshots.yml

```diff
@@ -73,24 +63,13 @@ jobs:
         mode: [dark, light]
     steps:
     - uses: actions/checkout@v1
-    - uses: actions/setup-ruby@v1
+    - name: "Set up Ruby"
+      uses: ruby/setup-ruby@v1
       with:
-        ruby-version: '2.6.x'
-    
-    - name: Install Bundler
-      run: gem install bundler
-    
-    - name: Restore Ruby Dependency Cache
-      id: restore-ruby-dependency-cache
-      uses: actions/cache@v1
-      with:
-        path: vendor/bundle
-        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-    
-    - name: Install Ruby Dependencies
-      run: |
-        bundle config path vendor/bundle
-        bundle install --jobs 4 --retry 3
+        bundler-cache: true
+
+    - name: Select Xcode Version
+      run: sudo xcode-select --switch /Applications/Xcode_12.app
```

Comment:
Haha I actually just learned that `xcversion` exists yesterday and didn't realize that it could be used here.

Changed in cfd0a24258937238af808732c8244279771aea31